src/instruments/src/EFB/Dashboard/Dashboard.tsx

```diff
@@ -52,7 +52,8 @@ const Dashboard: React.FC<DashboardProps> = (props) => {
                     </div>
 
                     <div className="h-3/5 ml-3 mt-3 rounded-2xl shadow-lg">
-                        <Map currentFlight={getCurrentFlightData} forceTileset="carto-dark" disableMenu hideOthers />
+                        {/* eslint-disable-next-line react/jsx-handler-names */}
```

Comment:
I think either we should enforce this rule and apply it everywhere, or we should remove the rule from the config. Or is there anything special about the usage here that warrants an exception?

Reply:
This is because:

1. a aprop with prefix `on` has to take a function prefixed with `on|handle`;
2. a prop being passed a function prefixed with `on|handle` must be prefixed with `on`.

The mistake here was re-naming `getCurrentFlightData` to `handleGettingCurrentFlightData,` which is incorrect since it is not an event handler.

I will fix this.